### ISSUE TITLE
Suggestion Help pop up window.

### ISSUE THREAD

**Issue by chrish13** [esjeon/krohnkite#96]
This is strictly a request or suggestion and not a bug.
May I please suggest to add if it's possible a pop up window with all the shortcuts Krohnkite has and also to be able to customize it according with user's personal (custom) shortcuts? 
Something like AwesomeWM has.

**Comment by esjeon** [comment]
Thanks for the suggestion. It should be possible by using QML features, but it can be tricky to figure out the actual key bindings set by user. I'll consider this.

**Comment by chrish13** [comment]
True..but you could also give the option to user to add more keys or if you can't then just the the keys that are assign by default from the krohnkite script or if the user customize those ones then just those.
